### DISCUSSION TITLE
usbipd.exe list -r ip_server -b x-x

### DISCUSSION THREAD

**Discussion by Giverson** [dorssel/usbipd-win#894] [General]
Hello everyone.
I tried to share the USB device using usbipd.exe on Windows 10 22h2, I got an attached device with the command usbipd.exe attach --wsl -b x-x, but when I tried to connect from the client using the command usbipd.exe attach -r server_ip -b x-x this command not exist, I trying use usbipd.exe list -r server_ip -b x-x and do not work.
Please, anyone tell me if I have something not correct?

**Comment by dorssel** [comment]
`usbipd-win` includes its own, dedicated, distribution-independent `usbip` client for WSL. It is in `C:\Program Files\usbipd-win\WSL`.
If you want to use `usbip` directly from within WSL, you will have to install it for your specific distribution, just like for any other Linux clients.

**Reply by Giverson** [reply]
Hi Dorssel.
I found this path here and found too the usbip in C:\Program Files\usbip-win\WSL but the file usbip as a archive here. I took 4.1.0 version. Sorry my question but it's correct the usbip in C:\Program Files\usbip-win\WSL  is an archive,I thinked that is a executable.

**Reply by dorssel** [reply]
You are not supposed to use that embedded `ubsip` yourself; it is for `usbipd-win` internal use only. You need to install a USBIP client for your distribution if you want to manually use the `usbip` command from within WSL.

**Reply by Giverson** [reply]
Sure, I understand. But in my case I will try to connect on the USB device from another host (Both Windows 10), To connect the remote from a client I would need to command usbip attach -r server_ip -b x-x (In Linux case). If the case is on Windows, how I do it, please? 
I just tried installing the wsl, usbip-win, and attaching the USB device on Windows host usbipd.exe attach --wsl -b x-x, after that, I don't go to the next step what would it be connect from another host Windows.

**Reply by dorssel** [reply]
See README:

> Note
> 
> Client-side tooling exists for other operating systems such as Microsoft Windows, but not as part of this project.

You will need to get Windows client side software from somewhere else.
